README.md

```diff
@@ -50,26 +61,41 @@ The `GameBoard` class is the core of the game logic and user interaction. It con
 - **`__init__(self, size)`**: Initializes the game board with a specified size and sets up the initial grid with empty cells.
 
 - **`initialize_empty_coordinates`**: Go troght all the grid as save the empty spaces in a set()
+   - Time Complexity: O(n^2) - where n is the size of the grid. This method iterates through each cell of the grid.
+
 
 - **`update(self)`**: Prints the current state of the game grid to the terminal.
+  - Time Complexity: O(n^2) - as it iterates through each cell for printing.
```

Comment:
if u changed the one before by O(n*m) change also here and where ever it is also needed

Reply:
I wont change both of them, I dont see it as necessary. Why would I specify n and m if they have to be equal ??

Reply:
Nice I agree